addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -233,6 +320,53 @@ weSnippetEditor.SnippetsMenu.include({
             throw e;
         }
     },
+    /**
+     * @private
+     */
+    _onAnimateTextClick(ev) {
+        const sel = window.getSelection();
+        const cursorPos = [sel.anchorNode, sel.anchorOffset, sel.focusNode, sel.focusOffset];
+        const editable = this.options.wysiwyg.$editable[0];
+        if (!sel.rangeCount) return;
+        const range = getDeepRange(editable, { splitText: true, select: true, correctTripleClick: true });
+        if (this.$currentAnimatedText.length) {
+            this.$currentAnimatedText.contents().unwrap();
+            setCursor(...cursorPos);
+            this._toggleHighlightAnimatedTextButton();
+            ev.target.classList.remove('active');
+            this.options.wysiwyg.odooEditor.historyStep();
+        } else {
+            if (sel.getRangeAt(0).collapsed) return;
```

Comment:
Convention

Reply:
Sorry... I saw this 50 times in web editor

Reply:
Yep, don't be contaminated :wink: